mobile-broadband-provider-info/serviceproviders.xml

```diff
@@ -4692,6 +4692,8 @@ conceived.
 			<apn value="everywhere">
 				<usage type="mms"/>
 				<name>MMS</name>
+				<username>eesecure</username>
+				<password>secure</password>
```

Comment:
This is so secure 🙂

Reply:
I know, secure password in plain text :P